Assoumpta/MyDiary
# ft(Connect database with my project)                                 …

What does this PR do?
- Insert the database with my project.

#### Description of Task to be completed
- A user should be able to signup using a form and the data be entered in the database.

#### How should this be manually tested?
- clone the repo and switch to the branch ft-database-168989236
- start the server
- use npm run create command
-Use npm start to see if the server is running on the port.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
As a user, I should be able to signup and sign-in on my account using my credentials.

#### Screenshots (if appropriate)
-N/A
